### PR TITLE
Gafi Milestone 1 PR

### PR DESCRIPTION
**Pull request by mutobui**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/896

### CODE REVIEW

File: deliveries/Gafi-milestone_1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Milestone Delivery :mailbox:
+
+* **The Invoice:** filled
+* **Application Document:** https://github.com/w3f/Grants-Program/pull/896
```
Comment (takahser) [inline deliveries/Gafi-milestone_1.md:4]:
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/Gafi.md
```
```suggestion
* **Application Document:** https://github.com/w3f/Grants-Program/pull/896
```

**Review by takahser — CHANGES_REQUESTED** [review]
The checks probably failed because you linked the PR instead of the application doc:)

**Comment by takahser** [comment]
@mutobui perfect, now the checks succeeded.
Anyway, thanks for the delivery. We will look into it (in more detail) as soon as possible. 👍

**Comment by mutobui** [comment]
> @mutobui perfect, now the checks succeeded. Anyway, thanks for the delivery. We will look into it (in more detail) as soon as possible. 👍

Thank you, excited to wait for your advice. We are willing to the changes.

**Review by Noc2 — APPROVED** [review]
I’m happy to tell you that the milestone is a pass. Keegan evaluated your delivery and was really happy with it: https://github.com/keeganquigley/Grant-Milestone-Delivery/pull/1 Everything immediately worked and the code looks good. I will forward your invoice internally.

**Comment by mutobui** [comment]
Thank you @Noc2 @takahser @keeganquigley and web3 foundation team for your relentless support, this first milestone will give us more resources to build Gafi Network and the web3 mission.

**Comment by RouvenP** [comment]
hi @mutobui we transferred the payment today. Thanks!

**Pull request merged**